site/src/layouts/DetailComponent/DetailComponent.module.css

```diff
@@ -22,4 +22,5 @@
 
 .content {
   min-width: 1px;
+  flex-basis: 800px;
```

Comment:
Is this magic number shared by something else? if so, maybe a css variable is needed?